src/main/java/gameframework/game/GameEntity.java

```diff
@@ -1,8 +1,7 @@
 package gameframework.game;
 
-import java.awt.Graphics;
+import gameframework.drawing.Drawable;
 
-public interface GameEntity {
-	public void draw(Graphics g);
+public interface GameEntity extends Drawable{
```

Comment:
The GameEntity interface is already defining a draw method though. Should I define the 2 draw methods in GameEntity instead, and not extend Drawable ?